tests/database/test_postgres.py

```diff
@@ -1,24 +0,0 @@
-import pytest
```

Comment:
I discovered this file was a no-op (there is a fixture, but no tests, so I just removed the whole thing)